src/omv/modules/py_fir.c

```diff
@@ -52,20 +54,23 @@
     __value & 0x87FF; \
 })
 
-static cambus_t fir_bus = {};
+cambus_t fir_bus = {};
 static void *fir_mlx_data = NULL;
 
 static enum {
     FIR_NONE,
     FIR_MLX90621,
     FIR_MLX90640,
-    FIR_AMG8833
+    FIR_AMG8833,
+#if defined(OMV_FIR_LEPTON_PRESENT)
+    FIR_LEPTON
+#endif
 } fir_sensor = FIR_NONE;
 
-static int fir_width = 0;
-static int fir_height = 0;
-static int fir_ir_fresh_rate = 0;
-static int fir_adc_resolution = 0;
+int fir_width = 0;
+int fir_height = 0;
+int fir_ir_fresh_rate = 0;
+int fir_adc_resolution = 0;
```

Comment:
Any way to keep these static ? Maybe put them in a struct and pass a pointer around ?